src/status_im/chat/views/message/message.cljs

```diff
@@ -75,7 +78,8 @@
           [react/icon icon-path style/command-image]])
        (if (:markup preview)
          ;; Markup was defined for command in jail, generate hiccup and render it
-         (commands.utils/generate-hiccup (:markup preview))
+         (cond-> (commands.utils/generate-hiccup (:markup preview))
+           network-mismatch? (conj [react/text send-network]))
```

Comment:
I know that our attempt at generic commands ended with failure and we practically only have `/send`/`/request` commands now, but it would be still nice to not tie/hardcode application code against specific commands.
Would it be possible to change to logic to use `:bot-db` for any state needed by bot (`transactor` in this case) and free application code from tight coupling with js code of particular bot ?

Reply:
@janherich we know about network mismatch only on receiver side, after command's markup was generated. I don't see way to compare network during markup generation on sender side.
I'm still have doubts about adding network name to command markup, maybe "network mismatch" warning is already enough to let user know something is wrong. If this is the case, we could safely remove this code.

Reply:
Actually, we are generating markup on the receiver side as well (`:preview`/`:short-preview`), so one way how to solve it would be to change the `previewRequest` function in the `transactor.js`, have it subscribe to `bot-db` and mirror actually selected network in `bot-db`.
All special logic of how to render the request could be then in the preview function in js code, `validateRequest` would also subscribe to that information in `bot-db` and you won't be able to actually send the request if you are on different network then what the request contains in params. 
I'm not sure if it's worth the time, but that's the proper way to actually "dogfood" our own command/response API and not sidestep it.

Reply:
@janherich thanks for clarifying. As this PR is close to be merged, i would prefer to create an separate issue for this.